db/documents.py

```diff
@@ -49,6 +72,8 @@ def __str__(self):
 
 document_type = sqla.Enum('oral', 'written', 'oral reexam', name='document_type', inherit_schema=True)
 
+class PaymentState:
+    NOT_ELIGIBLE, ELIGIBLE, DISBURSED = range(3)
```

Comment:
Does the distinction between `NOT_ELIGIBLE` and `DISBURSED` actually matter anywhere?